code/__HELPERS/global_lists.dm

```diff
@@ -21,7 +21,16 @@
 	//alt heads
 	init_sprite_accessory_subtypes(/datum/sprite_accessory/alt_heads, GLOB.alt_heads_list)
 
-	init_subtypes(/datum/surgery_step, GLOB.surgery_steps)
+	for(var/path in (subtypesof(/datum/surgery_step)))
+		var/datum/surgery_step/S = new path()
+		for(var/start_step in S.surgery_start_stage)
+			if(!S.name) // abstract subtype
+				continue
+			if(!GLOB.surgery_steps[start_step])
+				GLOB.surgery_steps[start_step] = list(S)
+			else
+				var/list/SS = GLOB.surgery_steps[start_step]
+				SS.Add(S)
```

Comment:
![image](https://user-images.githubusercontent.com/15887760/106734864-d04c3880-6613-11eb-8cdd-076e9df56479.png)
Won't work since it'll just assign it instead of make a list